app/models/mets_document/mods_document.rb

```diff
@@ -29,6 +29,30 @@ def title
           title
         end
       end
+      non_sort = value_from xpath: "mods:titleInfo/mods:nonSort"
+      return "#{non_sort} #{title_value}".gsub(/["\[\]]/, "") unless non_sort.empty?
+      title_value
+    end
+
+    def sort_title
+      normalize_whitespace(value_from(xpath: "mods:titleInfo/mods:title")).join(",")
```

Comment:
Can you make this only return if there's a nonSort element?

Reply:
ok